fk-testimonial/src/pages/PreviewTestimonialScreen/PreviewTestimonialScreen.js

```diff
@@ -2,19 +2,18 @@ import React, { useContext, useRef, useState } from "react";
 import { PlayFilledIcon, PencilIcon } from "../../assets/index";
 import ClientDetails from "../../components/ClientDetails/ClientDetails";
 import { TestimonialContext } from "../../context/TestimonialContext";
-import { useHistory } from "react-router-dom";
+import { setTestimonialUrl, setScreen } from "../../actions/action";
+import { THANK_YOU_SCREEN } from '../../constants'
 import "./style.css";
-import { setTestimonialUrl } from "../../actions/action";
 
 const PreviewTestimonialScreen = (props) => {
   const { state, dispatch } = useContext(TestimonialContext);
   const [playVideo, setPlayVideo] = useState(false);
   const { testimonialType } = props;
-  const history = useHistory();
   const videoRef = useRef(null);
 
   const onApproveClick = () => {
-    history.push("/ThankYouScreen");
+    dispatch(setScreen(THANK_YOU_SCREEN))
```

Comment:
same as abovedon't use extra function use it like dispatch({
type: THANK_YOU_SCREEN,
payload: data,
})